docs/monitoring/inventory.md

```diff
@@ -1,34 +1,31 @@
 title: Sematext Inventory Monitoring
 description: Sematext Inventory Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
 
-The [Sematext Agent](../agents/sematext-agent) provides a simple and versatile way of gathering machine-related information such as host, VM, or container properties, kernel versions, and installed packages. It presents them on a per-host basis allowing you not only to view the data but also search and compare different hosts. All of this, in the same place, shipped automatically and effortlessly, without any operational overhead.
+The [Sematext Agent](../agents/sematext-agent) provides a simple and versatile way of gathering machine-related information such as host, VM, or container properties, kernel versions, and installed packages. It presents them on a per-host, per-package or per-container-image basis allowing you not only to view the data but also search and compare different hosts. All of this, in the same place, shipped automatically and effortlessly, without any operational overhead.
 
-The Inventory information is available in the *Monitoring Infrastructure* section of the *Monitoring* tab of your Sematext Cloud account - your main place for hosts, virtual machines and containers information.
+The Inventory information is available in the *Inventory* section of the *Infrastructure* tab of your Sematext Cloud account - your main place for servers, packages and container images information.
 
 <!-- ![Sematext Inventory Main Screen](../images/monitoring/inventory_main_view.png) -->
 
 <video style="display:block; width:100%; height:auto;" controls autoplay loop>
-  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory.mp4" type="video/mp4" />
+  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory-group-by.mp4" type="video/mp4" />
 </video>
 
-### Inventory Info
+### Servers
+#### Info
 
-The consolidated infrastructure inventory provides insight into:
+Provides quick insight into how unified your servers are by showing:
 
-- Number of different packages in your monitored environment
+- Number of servers/VMs
 - Number of different OS distributions
 - Number of different OS versions
 - A number of different kernel versions
 
-![Inventory Info](../images/monitoring/inventory_inventory_info.png)
+#### Overview
 
-### Inventory Overview
+The Servers overview provides a per-host view of each server or virtual machine, along with the operating system and its version, number of CPUs, their type, memory, number of hard disk drives and total capacity.
```

Comment:
number of hard disk drives and total capacity => number of disk drives and their total capacity